.buildscript/bootstrap.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if ! which brew >/dev/null; then
```

Comment:
What if you're not on a Mac? It seems like a pretty arbitrary requirement for working on this package.